src/Ledger/Utxo.lagda

```diff
@@ -79,7 +79,7 @@ module _ (let open Tx; open TxBody) where
   outs tx = mapKeys (tx .txid ,_) (tx .txouts)
 
   balance : UTxO → Value
-  balance utxo = Σᵐᵛ[ x ← utxo ᶠᵐ ] getValue x
+  balance utxo = indexedSumᵐᵛ ⦃ Value-CommutativeMonoid ⦄ getValue (utxo ᶠᵐ)
 
   cbalance : UTxO → Coin
   cbalance utxo = coin (balance utxo)
```

Comment:
Did instance resolution become less useful in Agda 2.6.4, or is there another reason for these changes?

Reply:
This might be your fault 😂
https://github.com/agda/agda/issues/6364

Reply:
The real problem is of course that using bundled monoids as a typeclass was a bad idea from the start. I think we should switch this to an unbundled version and then I'd expect this to work again.

Reply:
Yes, that's the very issue. I think switching to unbundled representations is the way to go.